sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -3,479 +3,622 @@
+>;
+
 // @public
 export interface ReceivedMessageInfo extends SendableMessageInfo {
-    readonly _amqpMessage: AmqpMessage;
-    readonly deadLetterSource?: string;
-    readonly deliveryCount?: number;
-    readonly enqueuedSequenceNumber?: number;
-    readonly enqueuedTimeUtc?: Date;
-    readonly expiresAtUtc?: Date;
-    lockedUntilUtc?: Date;
-    readonly lockToken?: string;
-    readonly sequenceNumber?: Long;
+  readonly _amqpMessage: AmqpMessage;
+  readonly deadLetterSource?: string;
+  readonly deliveryCount?: number;
+  readonly enqueuedSequenceNumber?: number;
+  readonly enqueuedTimeUtc?: Date;
+  readonly expiresAtUtc?: Date;
+  lockedUntilUtc?: Date;
+  readonly lockToken?: string;
+  readonly sequenceNumber?: Long;
 }
 
 // @public
 export enum ReceiveMode {
-    peekLock = 1,
-    receiveAndDelete = 2
+  peekLock = 1,
+  receiveAndDelete = 2
 }
 
-// @public
-export class Receiver {
-    close(): Promise<void>;
-    getMessageIterator(): AsyncIterableIterator<ServiceBusMessage>;
-    get isClosed(): boolean;
-    isReceivingMessages(): boolean;
-    receiveDeferredMessage(sequenceNumber: Long): Promise<ServiceBusMessage | undefined>;
-    receiveDeferredMessages(sequenceNumbers: Long[]): Promise<ServiceBusMessage[]>;
-    receiveMessages(maxMessageCount: number, maxWaitTimeInSeconds?: number): Promise<ServiceBusMessage[]>;
-    get receiveMode(): ReceiveMode;
-    registerMessageHandler(onMessage: OnMessage, onError: OnError, options?: MessageHandlerOptions): void;
-    renewMessageLock(lockTokenOrMessage: string | ServiceBusMessage): Promise<Date>;
-    }
-
-export { RetryOptions }
+export { RetryOptions };
 
 // @public
 export interface RuleDescription {
-    action?: string;
-    filter?: string | CorrelationFilter;
-    name: string;
+  action?: string;
+  filter?: string | CorrelationFilter;
+  name: string;
 }
 
 // @public
 export interface RuleDetails {
-    action?: SqlAction;
-    createdOn: string;
-    filter?: SqlFilter | CorrelationFilter;
-    ruleName: string;
-    subscriptionName: string;
-    topicName: string;
+  action?: SqlAction;
+  createdOn: string;
+  filter?: SqlFilter | CorrelationFilter;
+  ruleName: string;
+  subscriptionName: string;
+  topicName: string;
 }
 
 // @public
 export interface RuleOptions {
-    action?: SqlAction;
-    filter?: SqlFilter | CorrelationFilter;
+  action?: SqlAction;
+  filter?: SqlFilter | CorrelationFilter;
 }
 
 // @public
 export interface SendableMessageInfo {
-    body: any;
-    contentType?: string;
-    correlationId?: string | number | Buffer;
-    label?: string;
-    messageId?: string | number | Buffer;
-    partitionKey?: string;
-    replyTo?: string;
-    replyToSessionId?: string;
-    scheduledEnqueueTimeUtc?: Date;
-    sessionId?: string;
-    timeToLive?: number;
-    to?: string;
-    userProperties?: {
-        [key: string]: any;
-    };
-    viaPartitionKey?: string;
+  body: any;
```

Comment:
JS: SendableMessageInfo
.NET: ServiceBusMessage
Java: SerciveBusMessage
Python: kwargs.

Before GA, we have to sit together and go through the public surface area.

Reply:
100% agreed on that one. 

Also..`ServivcBusMessage` is a typo, right? These comments should not be typed angrily....

Reply:
Issue filed: #7690